src/Microsoft.AspNetCore.Mvc.RazorPages/DependencyInjection/MvcRazorPagesMvcCoreBuilderExtensions.cs

```diff
@@ -52,6 +53,11 @@ public static IMvcCoreBuilder AddRazorPages(this IMvcCoreBuilder builder)
         internal static void AddServices(IServiceCollection services)
         {
             services.TryAddEnumerable(
+                ServiceDescriptor.Transient<
```

Comment:
Super hard to read with the lines broken up. Just put this entire bit on one line.